### PR TITLE
security.txt: create file

### PR DESCRIPTION
**Pull request by SMillerDev**
Detail how security researchers can contact us

### CODE REVIEW

**Comment by MikeMcQuaid** [comment]
Thanks @SMillerDev. Is this consumed by a particular tool?

**Comment by SMillerDev** [comment]
I'm sure it could be, but it's an RFC: https://securitytxt.org/

**Comment by Bo98** [comment]
Doesn't this need to be under `.well-known` or does GitHub Pages do that for us?

**Comment by Bo98** [comment]
Oh apparently both is valid, nevermind. `.well-known` is just the preferred one.

**Comment by carlocab** [comment]
> Oh apparently both is valid, nevermind. `.well-known` is just the preferred one.

Should we use `.well-known` given it's the preferred one?

**Comment by SMillerDev** [comment]
I don't know how well that works with GitHub's pages setup.

**Comment by MikeMcQuaid** [comment]
> I don't know how well that works with GitHub's pages setup.

Should work fine. Can check the output in `_site` when you run `jekyll build` locally.

File: security.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+layout: none
+permalink: .well-known/security.txt
```
Comment (MikeMcQuaid) [inline security.txt:3]: Does this work if you create an actual `.well_known` directory and stick it in there instead?

Comment (SMillerDev) [reply security.txt:3]: It doesn't pick up `.well-known`, probably because it sees it as a "hidden file"

Comment (MikeMcQuaid) [reply security.txt:3]: I think it should work if you add to `.include` like this: https://github.com/Homebrew/brew.sh/blob/master/_config.yml#L4

I really don't care particularly though so feel free to CBA my request 😁

Comment (SMillerDev) [reply security.txt:3]: Seeing how little I know about Jeckyll and that this works as expected I'd rather not bother with that.

**Review by MikeMcQuaid — APPROVED** [review]
Thanks @SMillerDev!

**Pull request merged**